guides/whitelist.htm

```diff
@@ -193,11 +212,31 @@ <h3>Greylist</h3>
                                       ❌
                                     {% endif %}
                                 </td>
+                                <td {% if project.gdpr == "yes" %}class="table-color-gdpr-yes"{% endif %}>
+                                    {% if project.gdpr == "yes" %}
+                                        <a href={{project.gdpr-enable-steps}}><span class="oi oi-warning"></span></a>
+                                    {% endif %}
+                                </td>
                             </tr>
                         {% endfor %}
                     </tbody>
                 </table>
             </div>
         </div>
+        <h3 id="about-gdpr" class="anchor-adjust">About GDPR</h3>
+            <p>GDPR is a law in the European Union for data privacy. Some projects have
+            interpreted it to mean that to you have to opt in for sharing any BOINC 
+            stats to other people. To earn Gridcoin you must enable this because
+            otherwise Gridcoin can't see how much work you are doing and reward it</p>
```

Comment:
repetitive

Reply:
Is a specific part repetitive? If it's the sharing stats and see how much work, that's not always obvious to new users that that's what it means. 

If you just mean that it's a bit lengthy I'd agree. How something like this?
``` GDPR is a law in the European Union on data privacy. Some projects have interpreted it to mean that to sharing BOINC stats to anyone requires opting in. Solo crunchers must enable it to earn Gridcoin because otherwise Gridcoin can't see how much work you've done and reward it```

Reply:
the repetition I meant was saying Gridcoin twice in one sentence.

Reply:
Not sure I can really get rid of it. I wouldn't want to use "it" here because that would be ambiguous. I'm using Gridcoin in two different sense here: the network as a whole and the Gridcoin you earn, so people might not connect that directly. I could say "be rewarded" but that would be more repetitive.

Reply:
Figured out a way to do this with:
`Solo crunchers must enable it to be rewarded because otherwise Gridcoin can't see how much work you've done and figure out how much you are owed`